examples/server/a_bit_of_everything.go

```diff
@@ -187,18 +186,8 @@ func (s *_ABitOfEverythingServer) UpdateV2(ctx context.Context, msg *examples.Up
 }
 
 // PatchWithFieldMaskInBody differs from UpdateV2 only in that this method exposes the field mask in the request body,
-// so that clients can specify their mask explicitly
+// so that clients can specify their mask explicitly. The difference is entirely in the http interface.
 func (s *_ABitOfEverythingServer) PatchWithFieldMaskInBody(ctx context.Context, request *examples.UpdateV2Request) (*empty.Empty, error) {
-	// low-effort attempt to modify the field mask to only include paths for the ABE struct. Since this is only for the
```

Comment:
This method could be removed entirely and the rest endpoint annotation in the proto file added an additional binding to the UpdateV2 method. But it's perhaps useful for documentation to have it separate. 🤷‍♂

Reply:
I think it'd be nicer to remove this and show that it's possible to use an additional binding instead. Would you mind changing it?

Reply:
Sure thing, I was very much on-the-fence about whether to keep it.